AboutMe.md

```diff
@@ -1,9 +1,9 @@
 ## About Me
-<p>Name:</p>
-<p>Preferred Name:</p>
-<p>Grade Level:</p>
-<p>Email:</p>
-<p>Computer Science Courses Taken:</p>
-<p>Computer Science Interest Areas:</p>
-<p>After High School Plans:</p>
+<p>Name: Derek Faulconer</p>
+<p>Preferred Name: Derek</p>
+<p>Grade Level: 11</p>
```

Comment:
Being a junior in this course I'm curious what type of computer science course you'd be interested in next year? I'm hoping I might be able to offer something more.